infor-design/enterprise
# 3918 - Process Indicator Enhancements

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Adds enhancements to process indicator to make it responsive, add labels and a content area and support more icons. In addition added a datagrid formatter.

**Related github/jira issue (required)**:
Fixes #3918

**Steps necessary to review your pull request (required)**:
- go to http://localhost:4000/components/processindicator
- look at the two new examples with icons
- test this in RTL, and all themes
- go to http://localhost:4000/components/processindicator/example-labels.html
- test this in RTL, and all themes
- resize and only one will be shown at a smaller breakpoint. labels will wrap 
- go to http://localhost:4000/components/datagrid/test-all-formatters.html
- see the new process indicator formatter
- try it in all themes/row sizes

**Included in this Pull Request**:
